pkg/mock/scheduledevents/internal/types/types.go

```diff
@@ -18,7 +18,7 @@ type Event struct {
 	Code              string `mapstructure:"code"`
 	Description       string `mapstructure:"description"`
 	State             string `mapstructure:"state"` // State of the scheduled event
-	EventID           string `mapstructure:"event-id"`
+	EventId           string `mapstructure:"event-id" json:"event-id"`
```

Comment:
the tag should be json:"eventId" and the property should be EventID unless I'm missing how this is processed.

Reply:
Addressed in latest commit